ui/ducks/send/send.js

```diff
@@ -2652,10 +2652,9 @@ export function updateSendAsset(
             providedDetails.address,
             sendingAddress,
             providedDetails.tokenId,
-          ).catch((error) => {
+          ).catch(() => {
             // prevent infinite stuck loading state
             dispatch(hideLoadingIndication());
-            throw error;
           })),
```

Comment:
This change wont affect native transfers;
It also wont affect ERC20 transfers because for ERC20 transfers `tokenCanBeTreatedAsAnERC20` [here](https://github.com/MetaMask/metamask-extension/blob/a1a62aa33572710ec915823d52427c93979ee622/app/scripts/metamask-controller.js#L4208) should be `true` making `details` defined and skipping the call to `getTokenStandardAndDetails`.
This should only affect NFT assets; 

This is throwing because the code detects that there is a missing property which is `symbol` in the case of the nfts i have tested with ([here](https://github.com/MetaMask/metamask-extension/blob/a1a62aa33572710ec915823d52427c93979ee622/ui/ducks/send/send.js#L2616)) and because the `providedDetails` has `standard` property it will go into the [else](https://github.com/MetaMask/metamask-extension/blob/a1a62aa33572710ec915823d52427c93979ee622/ui/ducks/send/send.js#L2649) and call `getTokenStandardAndDetails`.

The code is detecting that the `symbol` is missing because the nftDetails `symbol` is inside `nft.collection`,
but this piece of [code](https://github.com/MetaMask/metamask-extension/blob/a1a62aa33572710ec915823d52427c93979ee622/ui/ducks/send/send.js#L2616)
is expecting the symbol inside the nft object;

The properties that it is checking for an NFT are basically "name, symbol, tokenId" i dnt think tokenId can undefined for an NFT.
Looking at the number of users affected by this; i would expect that most of them if not all are missing symbol;
A missing symbol should not make the transaction fail; i have tested this and my transactions were successful.

We could either remove the throw error in the catch;

Or fix this while keeping the "throw" would be to sth similar to this;

`      let missingProperty = STANDARD_TO_REQUIRED_PROPERTIES[
        providedDetails.standard
      ]?.find((property) => {
        if (providedDetails.collection && property === 'symbol') {
          return providedDetails.collection[property] === undefined;
        }
        return providedDetails[property] === undefined;
      });`